static/templates/settings/organization_settings_admin.hbs

```diff
@@ -142,6 +142,18 @@
                   is_checked=realm_mandatory_topics
                   label=admin_settings_label.realm_mandatory_topics}}
 
+                <div class="input-group">
+                    <label class="checkbox">
+                        <input type="checkbox" class="realm_enable_read_receipts inline-block setting-widget prop-element" name="realm_enable_read_receipts" data-setting-widget-type="boolean" id="id_realm_enable_read_receipts" {{#if realm_enable_read_receipts}}checked="checked"{{/if}}/>
+                        <span></span>
+                    </label>
+                    <label for="id_realm_enable_read_receipts" class="inline realm_enable_read_receipts_label" id="id_realm_enable_read_receipts_label">
+                        {{#tr}}
+                            Enable read receitps (<i>Users can always disable their personal read receipts.</i>)
+                        {{/tr}}
+                    </label>
+                </div>
```

Comment:
We need the text in parens to be in italics. And we pass a string for label to `settings_checkbox`. I am not sure if we can do it in any other way.

Reply:
OK, that makes sense. Probably we could extend `settings_checkbox` template to support an `html_label` alternative field as a refactor to simplify this; seems better to avoid having duplicates of it scattered around.